velox/dwio/dwrf/test/TestColumnStatistics.cpp

```diff
@@ -602,50 +602,50 @@ TEST(StatisticsBuilder, basicMissingStats) {
 }
 
 TEST(StatisticsBuilder, basicHasNull) {
-  enum State { TRUE = 0, FALSE, MISSING };
+  enum class State { TRUE_STATE = 0, FALSE_STATE, MISSING_STATE };
```

Comment:
This code was copied from somewhere else. Per Velox coding standards the values should be kTrue, kFalse and kMissing. No need to add "State" suffix.

Reply:
Thank you @mbasmanova. I updated the code as you suggested:
```
enum class State { kTrue = 0, kFalse, kMissing };
```
Btw it seems most enum classes don't follow this standard. Will we be purging them later?

Reply:
> it seems most enum classes don't follow this standard. 

We need to fix these.

Reply:
We should also add a note about this in our coding style document.

Reply:
We already documented the use of "enum class". #838 documents the use of kPascalCase for the enumerators.